resourcemgr/resourcemgr.c

```diff
@@ -245,7 +246,7 @@ typedef struct {
                                     //  flushed.
     UINT64 connectionId;            // Used to identify which connection owns the object,
                                     // sequence, or session.
-    void *nextEntry; // Next entry in the list; 0 to terminate list.                                
+    RESOURCE_MANAGER_ENTRY_PTR nextEntry; // Next entry in the list; 0 to terminate list.                             
```

Comment:
There's also a pile of whitespace tacked on to the end of this line. It may just be an artifact from the line you've replaced but since you're fixing the void* thing you might as well do the whitespace cleanup at the same time.